src/components/atoms/RadioButton/RadioButton.test.tsx

```diff
@@ -0,0 +1,10 @@
+import { composeStories } from '@storybook/react';
+import { render } from 'src/test-utils.js';
+import * as stories from './RadioButton.stories';
+const { Default } = composeStories(stories);
+
+describe('RadioButton component', () => {
+    it('should render a RadioButton', () => {
```

Comment:
Please add tests for conditional styling and interaction with button which should cause a call to 'onChange' etc.